services/QuillLMS/app/lib/rule_feedback_history.rb

```diff
@@ -70,14 +70,16 @@ def self.generate_rulewise_report(rule_uid:, prompt_id:, start_date: nil, end_da
     }
   end
 
+  # rubocop:disable Metrics/CyclomaticComplexity
   def self.format_sql_results(relations)
     relations.map do |r|
+      first_feedback, second_feedback = r.feedbacks.sort_by { |f| f.order }
```

Comment:
You can use a pretzel colon here: `first_feedback, second_feedback = r.feedbacks.sort_by(&:order)`